plugin-template/android/Plugin.java

```diff
@@ -6,10 +6,10 @@
 import com.getcapacitor.PluginCall;
 import com.getcapacitor.PluginMethod;
 
-@NativePlugin()
+@NativePlugin
```

Comment:
Not related to the PR itself, but if the `()` is not really needed (tested and seems to work fine without it), we should probably remove it from all our core plugins, they all have it

Reply:
Prettier will remove them automatically as it did for the plugin template